www/html/ci/application/views/base.html

```diff
@@ -70,6 +70,19 @@
     </div>
     {% block content %}{% endblock %}
 
+
+    	<footer class="container-fluid center-block">
+    		<table><tr>
+    			<td><h6 id="copyright">Copyright KenTorii.com</h6></td>
+          {% if true == session %}
+    			<td><a type="button"><i class="fa fa-2x fa-unlock logout" onmouseover="lock(this)" onmouseout="unlock(this)" id="lock" aria-hidden="true"></i></a></td>
```

Comment:
Can you make the logout class an id instead here. Since we deleted the logout button Russell put, we only have one thing that has the logout capability. So we should make it an id.